scripts/quests/jeuno/LB06_New_Worlds_Await.lua

```diff
@@ -0,0 +1,145 @@
+-----------------------------------
+-- New Worlds Await
+-----------------------------------
+-- Log ID: 3, Quest ID: 133
+-- Nomad Moogle : !pos 10.012 1.453 121.883 243
+-----------------------------------
+require("scripts/settings/main")
+require("scripts/globals/items")
+require("scripts/globals/keyitems")
+require("scripts/globals/npc_util")
+require("scripts/globals/quests")
+require('scripts/globals/interaction/quest')
+local ID = require("scripts/zones/RuLude_Gardens/IDs")
+-----------------------------------
+local quest = Quest:new(xi.quest.log_id.JEUNO, xi.quest.id.jeuno.NEW_WORLDS_AWAIT)
+-----------------------------------
+
+quest.reward =
+{
+    fame  = 50,
+    fameArea = JEUNO,
+}
+
+quest.sections =
+{
+    -- Section: Quest available. Player doesn't have Limit Breaker KI.
+    {
+        check = function(player, status)
+            return status == QUEST_AVAILABLE and
+                player:getMainLvl() >= 75 and
+                player:getLevelCap() == 75 and
+                player:hasKeyItem(xi.ki.LIMIT_BREAKER) == false and
+                xi.settings.MAX_LEVEL >= 75
+        end,
+
+        [xi.zone.RULUDE_GARDENS] =
+        {
+            ['Nomad_Moogle'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(10045, 75, 2, 10, 7, 30, 302895, 4095)
+                end,
+            },
+
+            onEventFinish =
+            {
+                [10045] = function(player, csid, option, npc)
+                    if option ==  4 then
+                        npcUtil.giveKeyItem(player, xi.ki.LIMIT_BREAKER)
+                    end
+                end,
+            },
+        },
+    },
+
+    -- Section: Quest available. Got Limit Breaker KI. Era server section.
+    {
+        check = function(player, status)
```

Comment:
Just noticed this, but also occurs above.  Could you please use the full parameter list, even if vars isn't being used at the moment?
`check = function(player, status, vars)` for quests